pkg/vtpm/swtpm-vtpm/src/main.go

```diff
@@ -28,7 +28,7 @@ import (
 const (
 	swtpmPath      = "/usr/bin/swtpm"
 	maxInstances   = 10
-	maxPidWaitTime = 3 //seconds
+	maxPidWaitTime = 1 //seconds
```

Comment:
You mean you will only wait for one second for the swtpm process to start and write its pid file? When running tests in a virtual EVE that sounds very short.

Reply:
That is from before making the vTPM requests async (to not trigger watchdog), in the commit 80598891cf61e9c3ae0576870b478a3ce92eb8f1 changed it to 5 seconds.